uiowa/uiowa
# Admissions - Add fields to Areas of Study

Resolves https://github.com/uiowa/uiowa/issues/2741

## To Test

- First sync the default site and enable the Areas of Study feature split. https://default.local.drupal.uiowa.edu/admin/config/development/configuration/config-split
- `drush @default.local cim` to import the AoS config changes.
- The area of study feature should now be tracked in the sitemap and have the revisions limit set to 10.
- Next sync admissions. The AoS should already be turned on and the changes to the default AoS feature split as well as the fields listed in #2741 should all be present but un-styled (exception being the intro field which is using the body field instead but is also un-styled).
- Why Iowa should have a question mark on render but not on the form.
- Admissions' Student profile and admissions requirement content types should also have the sitemap and revisions tweaks.
- Admissions' student profile short blurb field should now be restricted to minimal/plain text.
- For the stories, you should see a card component template be used and a video or image can be added.
